src/tests/ngraph_helpers/lpt_ngraph_functions/src/gather_function.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "lpt_ngraph_functions/gather_function.hpp"
+
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include "lpt_ngraph_functions/common/builders.hpp"
+
+namespace ngraph {
+namespace builder {
+namespace subgraph {
+
+std::shared_ptr<ngraph::Function> GatherFunction::getOriginal(
+    const ngraph::PartialShape& inputShape,
+    const std::vector<size_t>& gatherIndicesShape,
+    const std::vector<int>& gatherIndicesValues,
+    const std::vector<int>& axis,
+    const int64_t batch_dims,
+    const ngraph::element::Type precisionBeforeDequantization,
+    const ngraph::builder::subgraph::DequantizationOperations& dequantization,
+    const int opset_version) {
```

Comment:
minor (thinking out loud, not required to change), one method for both cases with simple input arguments, will be reused three times:
```
std::shared_ptr<ngraph::Function> GatherFunction::get(
    // model params
    const ngraph::PartialShape& inputShape,

    // before operation
    const ngraph::element::Type precisionBefore,
    const ngraph::builder::subgraph::DequantizationOperations& dequantizationBefore,

    // operation
    const GatherParams& gatherParams,

    // after operation
    const ngraph::element::Type precisionAfter,
    const ngraph::builder::subgraph::DequantizationOperations& dequantizationAfter)
```

Reply:
I prefer to keep it as it is to keep to code style consistent with other files in the folder. Thank you!